### ISSUE TITLE
EO snippets in YAML packs are not checked for errors

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#530]
In YAML tests packs, we have EO snippets (`input` key in the single pack, and `*.eo` in the WPA pack). During the test run, we parse those snippets into XMIRs, in order to pass them as inputs to lints, and then we make an assertions about the lint result. Take a look at this example:

```yaml
sheets:
  - /org/eolang/lints/names/anonymous-formation.xsl
asserts:
  - /defects[count(defect[@severity='warning'])=0]
input: |
  # Main.
  malloc.of
    64
    [m] >>
      QQ.io.stdout > @
        "foo"
```
 
Let's say EO snippet (`input`) cannot be parsed (contains errors), however, test pack will pass. To make our test pipeline more robust, I suggest to check that each EO snippet does not contain any errors (in both scopes: single and WPA).

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by h1alexbel** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@h1alexbel good thinking, I agree!

**Comment by 0crat** [comment]
@h1alexbel Hey there, bug hunter! 🐛 Great catch on that new bug – you've just snagged yourself 12 points! Your eagle eye is helping us level up our project's quality. Got a hunch something's [not quite right](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) in the repo? [Don't hold back](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html), [let us know](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html)! Your current score's sitting pretty at [+219](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep those reports coming!
